network-api/networkapi/mozfest/templates/partials/primary_mozfest_nav.html

```diff
@@ -0,0 +1,70 @@
+{% load primary_active_nav %}
+
+<div id="primary-nav-container">
+  <div class="wrapper-burger">
+    <div class="menu-container">
+      <div class="narrow-screen-menu hidden d-lg-none">
+        <div class="narrow-screen-menu-background">
+          <div class="narrow-screen-menu-container">
+            <nav class="container">
+              <div class="row">
+                <div class="col">
+                  <div class="nav-links pt-3">
+                    <div><a class="{% primary_active_nav request '/' %}" href="/">Home</a></div>
```

Comment:
This code sort of exists "twice" in the partial, which threw me off the first time I edited this block and didn't see it update the page, only to realise I was editing the code that shows only at mobile widths. 

It might be a good idea to turn these links into an include as well, so that both navs include "the same links" but primed for a different UX role.

Reply:
https://github.com/mozilla/foundation.mozilla.org/issues/3160#issuecomment-493170721